### PR TITLE
Styled Canvas Elements

### PR DESCRIPTION
**Pull request by saadiqbal-dev**
image

### CODE REVIEW

File: src/components/Core.js [hunk 1]
```diff
@@ -159,7 +171,17 @@ const Core = (props) => {
 
   return (
     <CoreBackdrop className="core__auth">
-      <CoreBox className="classic">
+      <CoreBox
```
Comment (elliotBraem) [inline src/components/Core.js]: Can we move these styles out to App.scss?

File: src/components/Core.js [hunk 2]
```diff
@@ -185,17 +207,18 @@ const Core = (props) => {
                 src={"mob.near/widget/ProfileImage"}
                 props={{
                   accountId: account.accountId,
-                  className: "d-inline-block m-2",
-                  imageClassName: "rounded-circle w-100 h-100",
-                  style: { width: "42px", height: "42px" },
+                  className: "d-inline-block",
```
Comment (elliotBraem) [inline src/components/Core.js]: This too, the goal is for most of these styles to be configurable from the root

```css
.button {
   
}

core__auth: {
    .dropdown {
         .dropdown-item {
         
         }
    }
}
```

This auth layer (Core) is especially relevant. Think no-code -- different apps (like trylivepeer.near.page, etc) will be sharing this auth layer, but each will want to customize in their own way.

Comment (elliotBraem) [reply src/components/Core.js]: Last thing -- when thinking about extracting styles, when different apps implement their auth layer, they'll probably provide this as a widget (such as trylivepeer.near/widget/core) or something like that. So we only want to pull out *global* styles, that will be shared by these different widget implementations.

**Review by elliotBraem — COMMENTED** [review]
This looks great -- you've done well understanding a pretty *over*-complicated codebase. Left some suggestions to improve it.

File: src/components/Core.js (hunk 1 above)
Comment (saadiqbal-dev) [reply src/components/Core.js]: done

File: src/components/Core.js (hunk 2 above)
Comment (saadiqbal-dev) [reply src/components/Core.js]: done

**Comment by elliotBraem** [comment]
@saadiqbal-dev Checked it out locally, using Brave Browser on a Mac OS:

Overflow:
Screenshot 2024-04-09 at 5 48 04 PM

**Comment by saadiqbal-dev** [comment]
> @saadiqbal-dev Checked it out locally, using Brave Browser on a Mac OS:
> 
> Overflow: Screenshot 2024-04-09 at 5 48 04 PM

image
fixed the overflow

**Comment by elliotBraem** [comment]
How about tags like before?


Screenshot 2024-04-09 at 6 23 44 PM

**Comment by saadiqbal-dev** [comment]
image

fixed boss

**Pull request merged**